packages/components/src/components/contract-card/contract-card-items/contract-card-header.jsx

```diff
@@ -20,20 +21,37 @@ const ContractCardHeader = ({
     id,
     is_mobile,
     is_sell_requested,
+    is_sold: is_contract_sold,
     is_valid_to_sell,
     onClickSell,
     server_time,
 }) => {
     const current_tick = contract_info.tick_count ? getCurrentTick(contract_info) : null;
-    const { underlying, multiplier, contract_type, shortcode, purchase_time, date_expiry, tick_count, is_sold } =
-        contract_info;
+    const {
+        growth_rate,
+        underlying,
+        multiplier,
+        contract_type,
+        shortcode,
+        purchase_time,
+        date_expiry,
+        tick_count,
+        tick_passed,
+    } = contract_info;
+    const is_sold = !!contract_info.is_sold || is_contract_sold;
+    const is_accumulator = contract_type === 'ACCU';
```

Comment:
I suggest not to hardcode the type, you can set it at one place and reuse it because it is hardcoded in multiple files atm